api-scripts/patron/placing-holds/jmeter-supported-data/instances_without_requests.csv

```diff
@@ -0,0 +1,1220 @@
+00000eac-23fc-44de-8ac4-ec7aa032e263
```

Comment:
Can you explain in the README why we need instances without requests? (I may guess that it's to make cleaning up easily, but please describe its role.)